build/yamls/kafka-server.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
```

Comment:
Is this taken from any open source project? Should we add a reference?

Reply:
I created from scratch using deployment template. So no need to add special reference I think.